jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -691,6 +697,9 @@ frontend wss-in
   <%- if properties.ha_proxy.frontend_config -%>
     <%= format_indented_multiline_config(p("ha_proxy.frontend_config")) %>
   <%- end -%>
+  <% if p("ha_proxy.expect_proxy") -%>
```

Comment:
add a check in the frontend spec test that this config line is output when you have the expected configuration.